tests/profile/test_profile_against_externals.py

```diff
@@ -0,0 +1,78 @@
+""" Profiles basic -jX functionality """
+# Copyright (c) 2020 Frank Harrison <doublethefish@gmail.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+# pylint: disable=protected-access,missing-function-docstring,no-self-use
+
+import os
+import pprint
+import shutil
+import tempfile
+
+import pytest
+
+from pylint.lint import Run
+from pylint.testutils import TestReporter as Reporter
+
+
+class TempdirGuard:
```

Comment:
pytest has a built in fixture for doing this (https://docs.pytest.org/en/stable/tmpdir.html). Can we use that instead?